internal/services/telegramservice.go

```diff
@@ -0,0 +1,138 @@
+package services
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"telebot/internal/models"
+	"telebot/internal/utils"
+)
+
+// Запрос обновлений
+func GetUpdates(token string, offset int) ([]models.Update, error) {
+	requestUrl := utils.TelegramApiUrl + token + "/getUpdates" + "?offset=" + strconv.Itoa(offset)
+	resp, err := http.Get(requestUrl)
+	if err != nil {
+		return nil, utils.CommonError(err.Error() + " " + requestUrl)
+	}
+	defer resp.Body.Close()
+
+	bodyJson, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, utils.CommonError(err.Error() + " " + requestUrl)
+	}
+
+	if utils.IsFailStatus(resp.StatusCode) {
+		return nil, utils.HttpError(resp.StatusCode, resp.Status+" "+requestUrl+" "+string(bodyJson))
+	}
+
+	var getUpdateResponse models.GetUpdateResponse
+	err = json.Unmarshal(bodyJson, &getUpdateResponse)
+	if err != nil {
+		return nil, utils.CommonError(err.Error() + " " + requestUrl)
+	}
+
+	return getUpdateResponse.Result, nil
+}
+
+func SendTextMessage(token string, chatId int, text string) error {
+	requestUrl := utils.TelegramApiUrl + token + "/sendMessage"
+
+	message := models.BotMessage{
+		ChatId:      chatId,
+		Text:        text,
+		ReplyMarkup: models.ReplyKeyboardRemove{RemoveKeyboard: true},
+	}
+
+	messageJson, err := json.Marshal(message)
+	if err != nil {
+		return utils.CommonError(err.Error() + requestUrl + " " + fmt.Sprintf("%v", message))
```

Comment:
Тогда можно все внести в `fmt.Sprintf`:

```suggestion
		return utils.CommonError(fmt.Sprintf("%v: %s %v", err, requestUrl, message))
```

Reply:
забыл сделать, спс поправил)